app/Http/Requests/StoreStation.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreStation extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+    /**
+     * Get the validation rules that apply to the request.
+     */
+    public function rules(): array
+    {
+        return [
```

Comment:
Advertising myself: last year I published small package to handle validation rules: https://github.com/krzysztofrewak/laravel-oop-validator You can check it out, maybe it would be cool to use. ;)